src/assets/styles/abstracts/_variables.scss

```diff
@@ -77,3 +18,52 @@ header {
     --ildh-wordmark-colour: var(--fl-bgColor, #fff);
     --ildh-wordmark-text-colour: var(--fl-fgColor, #000);
 }
+
+body {
```

Comment:
It's best practice for all general-purpose custom properties to be attached to `:root` rather than `body`.